src/test/test-seccomp.c

```diff
@@ -134,6 +134,7 @@ static void test_filter_sets(void) {
 }
 
 static void test_restrict_namespace(void) {
+#if !SECCOMP_RESTRICT_ADDRESS_FAMILIES_BROKEN
```

Comment:
Shouldn't that ifdef be added to test_restrict_address_families() instead of test_restrict_namespace()